vividus-extension-selenium/src/main/resources/spring.xml

```diff
@@ -138,13 +138,7 @@
 
     <bean class="org.vividus.ui.action.search.IndexElementFilter" />
 
-    <bean class="org.vividus.selenium.screenshot.AbstractScreenshotTaker" id="abstractScreenshotTaker" abstract="true">
-        <property name="screenshotDirectory">
-            <bean class="java.io.File">
-                <constructor-arg value="${ui.screenshot.directory}" />
```

Comment:
it makes sense to remove the default property as well: https://github.com/vividus-framework/vividus/blob/b329f3c1ffdd5d71e40f3f70800f76d08a789d6e/vividus-extension-selenium/src/main/resources/properties/defaults/default.properties#L25